rll/setup_software_repo.sh

```diff
@@ -92,22 +60,7 @@ EOF
 # AWS-Linux
 #
 elif [[ `cat /etc/system-release` =~ ^Amazon\ Linux.*\ ([0-9]+)\. ]]; then
-  ver="${BASH_REMATCH[1]}"
-  case "$ver" in
-  2014)
-    sudo dd of=/etc/yum.repos.d/RightScale-Software.repo <<EOF
-[rightscale]
-name=RightScale
-baseurl=http://mirror.rightscale.com/rightscale_software/centos/7/x86_64
-gpgcheck=1
-gpgkey=http://mirror.rightscale.com/rightlink/rightscale.pub
-enabled=1
-priority=5
-EOF
-    time sudo yum -y install yum-plugin-security
-    # update-minimal fails on RHEL7, see https://bugzilla.redhat.com/show_bug.cgi?id=1048584
-    time sudo yum -y --security update
-    ;;
-  esac
-
+  time sudo yum -y install yum-plugin-security
+  # update-minimal fails on RHEL7, see https://bugzilla.redhat.com/show_bug.cgi?id=1048584
+  time sudo yum -y --security update
 fi
```

Comment:
Can we have an `else` section logging that unable to determine OS or packaging software, no updates installed?